tests/test_core.py

```diff
@@ -32,20 +33,18 @@ def test_check_window_size():
 def naive_compute_mean_std(T, m):
```

Comment:
I have a concern that this function is no longer "naive" and is actually a copy of what we are testing against.